src/BaseServiceProvider.php

```diff
@@ -111,6 +113,11 @@ public function registerAdminMiddleware(Router $router)
         }
     }
 
+    public function registerCustomAuthGuard()
+    {
+        View::composer('backpack::*', \Backpack\Base\app\Http\ViewComposers\AuthComposer::class);
```

Comment:
@jorenvanhee - I'm wondering if this is broad enough. Wouldn't this make the $backpackAuth guard available only in views that are loaded within the backpack namespace?

If so, I see the following problem:

developer uses $backpackAuth when he overwrites backpack views;
developer create supplementary views, that he considers to be part of the admin panel; he tries to use $backpackAuth there too; he expects $backpackAuth to be there too, but it's not;
I may be mistaken, though. It's been known to happen :-)